rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtAuthenticationEntryPoint.java

```diff
@@ -1,13 +1,12 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.security.jwt;
 
-import org.springframework.security.core.AuthenticationException;
-import org.springframework.security.web.AuthenticationEntryPoint;
-import org.springframework.stereotype.Component;
-
+import java.io.IOException;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+import org.springframework.stereotype.Component;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'org\.springframework\.stereotype\.Component' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.